map/mobile-map-search-and-route/src/main/java/com/esri/samples/mobile_map_search_and_route/MobileMapSearchAndRouteSample.java

```diff
@@ -90,7 +87,7 @@ public void start(Stage stage) {
       mapPackageListView.setMaxSize(170, 100);
 
       // create a file chooser to find local mmpk files
-      FileChooser fileChooser = new FileChooser();
+      var fileChooser = new FileChooser();
       FileChooser.ExtensionFilter mmpkFilter = new FileChooser.ExtensionFilter("Map Packages (*.mmpk)", "*.mmpk");
       fileChooser.getExtensionFilters().add(mmpkFilter);
       fileChooser.setInitialDirectory(new File(System.getProperty("data.dir"), "./samples-data/mmpk"));
```

Comment:
It would probably be better to use `Path.resolve` etc. here because you'e hardcoded the path separator and I'm sure in other places it uses something to avoid doing that (https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/io/File.html#separatorChar maybe).